Misc/Concert/bd_010030.csv

```diff
@@ -0,0 +1,41 @@
+name01#0,"""？？？"""
+name02#0,"""？？？"""
+un_015180_0005#0,"""¡Gracias a todos por esperar!"""
+un_015180_0010#0,"""¡Vamos a toda velocidad! Asi que asegúrense de mantener el ritmo!"""
+un_015180_0020#0,"""La galaxia brillante se eleva a través del espacio! Todos brillan como estrellas, 『Cosmic twinkle star』!"""
+bd_010030_0010#0,"""Desfile cósmico en medio de un espacio resplandeciente<br><br><blu>cosmic parade Memagurushi space<c>"""
+bd_010030_0020#0,"""Estrellas que se reencarnan en millones de años luz<br><br><blu>Nan man nan noku kowonen boshi no Reincarnation<c>"""
+bd_010030_0030#0,"""Deslizándose del pasado al futuro como una brillante montaña rusa<br><br><blu>Kako kara mirai e kasowo suru   Shining Jetcoaster<c>"""
+bd_010030_0040#0,"""Transmitir innumerables esperanzas y sueños<br><br><blu>Ikutsu no kibo no sete<c>"""
+bd_010030_0050#0,"""Mirar hacia el futuro, y da un paso adelante para alcanzar nuestros sueños<br><br><blu>Hora mi a gete tachidoma tate hajimaranai yo o date<c>"""
+bd_010030_0060#0,"""Un mundo tan bueno e infinito está esperando<br><br><blu>Kon na sutekina mu gen no sekai ga ma a teru no<c>"""
+bd_010030_0070#0,"""Vamos a zarpar entre el mar de estrellas parpadeantes<br><br><blu>Ginga o watare twinkle star saa kake nukeyou<c>"""
+bd_010030_0080#0,"""Y tener la aventura de nuestras vidas<br><br><blu>Kira meku y y shunno nando mo kakina gara<c>"""
+bd_010030_0090#0,"""Deja que la futura canción se escuche más allá del horizonte<br><br><blu>Hibi kasete future song haruka kanata made<c>"""
+bd_010030_0100#0,"""Hacia la gran aventura<br><br><blu>Biigusukeru no adobencha<c>"""
+bd_010030_0110#0,"""¡Vamos a despegar!<br><br><blu>Min na de tobiko mou yo<c>"""
+bd_010030_0120#0,"""Los desconocidos Flashing Skies serán nuestro escenario<br><br><blu>Flashing sky michi o himeta stage<c>"""
+bd_010030_0130#0,"""Un misterioso destino lleno de brillantes joyas<br><br><blu>Hoseki o maki chirashita shinpi no destination<c>"""
+bd_010030_0140#0,"""Las proyecciones del polvo de estrellas Merry-go-round<br><br><blu>Horoguramu no keshiki ga mawaru Stardust Merry-Go-Round<c>"""
+bd_010030_0150#0,"""Hacia lo desconocido<br><br><blu>Mi taiken no uzu we<c>"""
+bd_010030_0160#0,"""Sigue adelante sin dudas<br><br><blu>Mayoi toma dou toki mo ushiro o furimui tari shinaide<c>"""
+bd_010030_0170#0,"""Siempre el mañana brilla ante mis ojos<br><br><blu>Itsumo me no mae ni ka ga ya ku ashita wa hirogaru no<c>"""
+bd_010030_0180#0,"""La galaxia de estrellas parpadeantes ilumina nuestros sueños"<br><br><blu>Ginga o megure twinkle star yume o terashi da se<c>""
```

Comment:
Missing a `"` at the end of this line

Reply:
repair what the  " and the error continues

Reply:
linea 23 faill